package.json

```diff
@@ -16,5 +17,18 @@
   "bugs": {
     "url": "https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/issues"
   },
-  "homepage": "https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations#readme"
+  "homepage": "https://twin-cities-mutual-aid.org/",
+  "devDependencies": {
+    "@babel/preset-env": "^7.10.2",
+    "@snowpack/plugin-babel": "^1.0.0",
+    "@snowpack/plugin-webpack": "^1.1.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
+    "snowpack": "^2.2.2"
+  },
+  "dependencies": {
+    "list.js": "^1.5.0",
```

Comment:
You left this on CDN, right? Do you need it here, too?

Reply:
I got it vendored instead. Should work now.

Reply:
I switched it to the vendored dependency and removed the CDN reference.